client/src/pages/login.tsx

```diff
@@ -49,6 +49,7 @@ export const Login: React.FC = () => {
   return (
     <Container
       style={{
+        backgroundColor: '#F2F2F2',
```

Comment:
The background color is changed only on the container and it hasn't changed for the entire screen, like in the pic below. And the name should state Car Rental Services in place of Refine Project.
<img width="2240" alt="Screenshot 2024-03-19 at 16 51 07" src="https://github.com/DevsRide/mern-car-rent-app/assets/11690402/b067edc5-d654-4059-9f3b-78af93998d11">

Reply:
The Powered by Google paragraph is removed successfully though, congrats!